File: Algoritmo(C)/Alg_S6_10.c
#include <stdio.h>
int main(){
    //variables
    int idade;
    //input
    printf("Digite a idade do nadador: ");
    scanf("%d",&idade);
    //process + output
    if (idade >= 18){
        printf("Adulto");
    }else if (idade >= 14){
        printf("Juvenil-B");
    }else if (idade >= 12){
        printf("Juvenil-A");
    }else if (idade >= 8){
        printf("Infantil-B");
    }else if (idade >= 5){
        printf("Infantil-A");
    }
}

File: Algoritmo(Python)/Alg_S6_10.py
#input
idade = int(input("Digite a idade do nadador: "))
#process
if(idade >= 18):
    print("Adulto")
elif(idade >= 14):
    print("Juvenil-B")
elif(idade >= 12):
    print("Juvenil-A")
elif(idade >= 8):
    print("Infantil-B")
elif(idade >= 5):
    print("Infantil-A")